src/app/chat/views/suggestions_list.nim

```diff
@@ -25,14 +26,15 @@ QtObject:
     result.setup
 
   proc rowData(self: SuggestionsList, index: int, column: string): string {.slot.} =
-    if (index >= self.suggestions.len - 1):
+    if (index >= self.suggestions.len):
```

Comment:
This was a pain in the ass. I couldn't understand why one of my mention was empty in QML, that was it